src/bridge/lib/versions.py

```diff
@@ -6,5 +6,6 @@
 KEYCLOAK_VERSION = "latest"
 REDASH_VERSION = "9d273e4"
 TRAEFIK_VERSION = "2.9.1"
+TUTOR_PERMISSIONS_VERSION = "15.3.4"
```

Comment:
I'm honestly not sure if we need to do this permissions thing but tutor did it so I retained it. This is like an init container that fixes ownership on a couple directories prior to starting edxapp-proper.